datasets/jokes.py

```diff
@@ -54,7 +54,10 @@ def generate_examples(self, num_examples):
                 selected_jokes.append(joke)
                 is_question.append(False)
                 time_jump = create_time_jump(self.minutes_low, self.minutes_high)
-                waits.append(WaitCreator.create_wait(tokens=randint(self.filler_tokens_low, self.filler_tokens_high), time=time_jump))
+                waits.append(WaitCreator.create_wait(
+                    tokens=randint(self.filler_tokens_low, self.filler_tokens_high),
```

Comment:
Should we remove the random filler? I don't think we have ever used it

Reply:
Yeah, let's take the chance.